pkg/controller/config/config_controller.go

```diff
@@ -94,6 +95,9 @@ func (a *Adder) InjectProcessExcluder(m *process.Excluder) {
 	a.ProcessExcluder = m
 }
 
+func (a *Adder) InjectMutationCache(mutationCache *mutation.System) {
```

Comment:
nit: this went back to being two lines, this comment applies in multiple places

Reply:
Fixed